boost-maven/boost-maven-plugin/src/main/java/io/openliberty/boost/maven/utils/MavenProjectUtil.java

```diff
@@ -51,4 +55,42 @@ public static String findSpringBootVersion(MavenProject project) {
         return dependencies;
     }
 
+    public static String getJavaCompilerTargetVersion(MavenProject project) {
+
+        Properties mavenProperties = project.getProperties();
+
+        for (Map.Entry<Object, Object> property : mavenProperties.entrySet()) {
+            String propertyKey = property.getKey().toString();
+            String propertyValue = property.getValue().toString();
+
+            if (propertyKey.equals("maven.compiler.target") || propertyKey.equals("maven.compiler.release")) {
+                return propertyValue;
+            }
+        }
+
+        List<Plugin> plugins = project.getBuildPlugins();
+
+        for (Plugin plugin : plugins) {
```

Comment:
Refactor the plugin configuration retrieval to use [`MavenProjectUtil`](https://github.com/WASdev/ci.maven/blob/master/liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/utils/MavenProjectUtil.java) instead.

Reply:
Nice - will do.